examples/basics/visuals/volume_plane.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+# vispy: gallery 2
+"""
+Example volume rendering with plane rendering
+
+Controls:
+* 1  - toggle between volume rendering methods
+* 2 -  toggle between volume rendering modes ('volume', 'plane')
+* [] - shift along plane normal
+* {} - decrease/increase plane thickness
+
+* x/y/z/o - set plane normal along x/y/z or [1,1,1] oblique axis
+"""
+import numpy as np
+
+from vispy import app, scene, io
+from vispy.visuals.transforms import STTransform
+
+# Read volume
+vol = np.load(io.load_data_file('volume/stent.npz'))['arr_0']
+
+# Prepare canvas
+canvas = scene.SceneCanvas(keys='interactive', size=(800, 600), show=True)
+
+# Set up a viewbox to display the image with interactive pan/zoom
+view = canvas.central_widget.add_view()
+
+# Create the volume visual
+volume = scene.visuals.Volume(vol, parent=view.scene, raycasting_mode='plane')
+volume.transform = scene.STTransform(translate=(64, 64, 0))
+
+# Create a camera
+fov = 60.
+cam = scene.cameras.TurntableCamera(parent=view.scene, fov=fov, name='Turntable')
+
+view.camera = cam
+
+# Create an XYZAxis visual
+axis = scene.visuals.XYZAxis(parent=view)
+s = STTransform(translate=(50, 50), scale=(50, 50, 50, 1))
+affine = s.as_matrix()
+axis.transform = affine
+
+
+# Implement axis connection with cam2
+@canvas.events.mouse_move.connect
+def on_mouse_move(event):
+    if event.button == 1 and event.is_dragging:
+        axis.transform.reset()
+
+        axis.transform.rotate(cam.roll, (0, 0, 1))
+        axis.transform.rotate(cam.elevation, (1, 0, 0))
+        axis.transform.rotate(cam.azimuth, (0, 1, 0))
+
+        axis.transform.scale((50, 50, 0.001))
+        axis.transform.translate((50., 50.))
+        axis.update()
```

Comment:
Is there no way to do this with the builtin functionality of the axis and/or camera?

Reply:
Probably! I actually ripped this out of the existing volume example, happy to try to update both in a followup PR?

Reply:
I'm ok with that. I'd like it in this PR, but no reason to hold it up if you can't get it...and I just realized github says the code is outdated.

Reply:
Cool that's fair! I'm not immediately sure how to do it so it will take me some time but happy to spend the time to make the PR better